pytorch_lightning/plugins/training_type/tpu_spawn.py

```diff
@@ -130,7 +130,8 @@ def barrier(self, name: Optional[str] = None) -> None:
             rendezvous(f"pl.Trainer.{name}")
 
     def transfer_distrib_spawn_state_on_fit_end(self, results):
-        best_model_path = self.lightning_module.trainer.checkpoint_callback.best_model_path
+        checkpoint_callback = self.lightning_module.trainer.checkpoint_callback
+        best_model_path = checkpoint_callback.best_model_path if checkpoint_callback else None
```

Comment:
what happens if there are multiple checkpoint callbacks attached?

Reply:
I think I'd rather use the first `best` path and log the path used when running `test`